javascript/net/grpc/web/grpcwebclientbase.js

```diff
@@ -272,11 +279,11 @@ class GrpcWebClientBase {
             message: 'Incomplete response',
           });
         } else {
-          callback(null, responseReceived);
+          callback(null, responseReceived, null, null, /* unaryResponseReceived= */ true);
```

Comment:
this is where things get a little bit weird.. since in `rpcCall` we're directly taking a user-provided callback.. so this implementation is a bit "leaky".. although, probably (hopefully) not a big deal.. 😄

Reply:
oh good point. I'll open a follow up not to pass the extra flag when useUnaryResponse is off

Reply:
#1297

Reply:
ah! thanks very much for the follow-up fix! 😃